setup.py

```diff
@@ -94,9 +95,10 @@ def make_abs_path(l):
 supported_frameworks = {
     "all": all_sources,
     "pytorch": pytorch_sources,
+    "jax": None, # JAX use transformer_engine/CMakeLists.txt
 }
 
-framework = "all"
+framework = os.environ.get("FRAMEWORK", "all")
```

Comment:
I would actually set that to pytorch for now in order to not break the existing workflows. We can switch that to all once the JAX support is fully merged.

Reply:
Done